src/executors/helm-chart-testing.yaml

```diff
@@ -0,0 +1,3 @@
+---
+docker:
+  - image: quay.io/helmpack/chart-testing:v2.4.0
```

Comment:
Why do we need a separate executor here?

Reply:
Helm chart testing tool isn't apart of architect image. So unless the command/job includes steps to install it, we can't use the existing executor.  I felt, using this executor would be simpler than replicating what it does in a command/job. 

I was intending to raise PR against architect repo to include them but wasn't really if it would be needed.

Reply:
Yeah I figured later and forgot to delete this comment. I think it's ok to use the image. It should be retagged but I left another comment for that.